src/styles/blooks/card.scss

```
@@ -0,0 +1,111 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: calc(200px - 34px);
+  width: 100%;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 16px;
+  background-color: $white;
+  border-radius: 5px;
+  border: 1px solid $light-gray;
+  transition: transform .3s linear;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    height: 134px;
+    max-width: 160px;
+    margin: 16px auto 0;
+  }
+
+
+  &__info {
+    margin-top: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    color: $darkblue;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color .3s linear;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__subtitle {
+    color: $gray;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 4px 0 0;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    color: $darkblue;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-top: 24px;
+    color: #616070;
```

don't forget to create variables for all colors
```suggestion
    color: ...;
```